oxide-auth/Cargo.toml

```diff
@@ -14,8 +14,10 @@ license = "MIT OR Apache-2.0"
 autoexamples = false
 
 [dependencies]
+argonautica = "0.2.0"
 base64 = "0.11"
 chrono = "0.4.2"
+lazy_static = "1.4.0"
```

Comment:
I'd prefer [`once_cell`](https://crates.io/crates/once_cell) here. It's the same effect without macro. The compiler is also currently switching to it in their implementation.

Reply:
Nice, didn't know that one.

Reply:
Fixed